src/style.scss

```diff
@@ -0,0 +1,10 @@
+@import "variables";
+@import "product-card";
+@import "product-card-photo";
+@import "product-card-article";
+@import "product-card-code";
+@import "product-card-rating";
+@import "product-card-review";
+@import "product-card-price";
+@import "product-card-buy";
```

Comment:
I think that you don't have to separate everything here, because it is one block and it's items. If you will write everything in one file, you can use scss nesting:
```scss
.product-card {
  ...
  
  &__rating {
    ...
  }

  &__code {
    ...
  }
}
```